website/docs/help-and-support/troubleshooting-guide/query-errors.md

```diff
@@ -5,6 +5,16 @@ description: >-
 # Query Errors
 The section illustrates common query errors and how to resolve them on Appsmith.
 
+## Find error messages from the datasource
```

Comment:
Under the troubleshooting section, only explanation related to actual issues and errors encountered should be documented. 

IMO, this information should be documented in the Queries doc - https://docs.appsmith.com/core-concepts/data-access-and-binding/querying-a-database/ 

Maybe a new section below 'Using Queries in applications' titled 'Debugging queries'

Reply:
Moved to the suggested page, definitely makes sense as it should be general-purpose knowledge